x-pack/plugins/apm/public/components/app/metrics/index.tsx

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import { isJavaAgentName, isJRubyAgent } from '../../../../common/agent_name';
+import { useApmServiceContext } from '../../../context/apm_service/use_apm_service_context';
+import { ServiceMetrics } from './service_metrics';
+import { ServiceNodeOverview } from './service_node_overview';
```

Comment:
Isn't this jvm specific? I think we should make that clear in the name:

```suggestion
import { JvmMetricsOverview } from './jvm_metric_overview';
```

Reply:
In the same vain, you should also rename the backend APIs and files:

`x-pack/plugins/apm/server/routes/service_nodes/route.ts` -> `x-pack/plugins/apm/server/routes/metrics/route.ts`

There's also the API:
`GET /internal/apm/services/{serviceName}/serviceNodes` which should be renamed to something like:
`GET /internal/apm/services/{serviceName}/metrics/nodes`